openexr-rs/src/deep/mod.rs

```diff
@@ -0,0 +1,48 @@
+pub use openexr_sys::DeepImageState;
```

Comment:
I think I prefer what you did with core. Where you listed all the modules first and then all of the 'use' after. For me was easier to scan.